deploy-agent/deployd/common/single_instance.py

```diff
@@ -17,19 +17,22 @@
 import logging
 import os
 import stat
-import tempfile
+import errno
 import fcntl
 from . import utils
+from future.utils import PY3
 
 log = logging.getLogger(__name__)
+LOCKFILE_DIR = '/var/lock'
```

Comment:
maybe add a note that it must be an absolute path for python 2 or os.makedirs will become confused

Reply:
I actually added a comment regarding `exist_ok` for py2.